src/interfaces/ICentrifugeRouter.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.5.0;
+
+interface ICentrifugeRouter {
```

Comment:
I'm just curious what do we need this interface for? Is it so that it's easier for 3rd parties to integrate?  They just import the interface  in their code  and does something like:
```
ICentrifugeRouter(address).requestDepost(...)
```

Reply:
Reason number 1 is that we have interfaces for all contracts such that we can have natspec documentation in the interfaces, and then use `inheritdoc` tags in the contracts.

Second reason is exactly what you said!